easybuild/easyconfigs/p/Pindel/Pindel-0.2.5b9-20170508-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,57 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+#
+# Author: Jonas Demeulemeester
+# The Francis Crick Insitute, London, UK
+##
+
+easyblock = 'CmdCp'
+
+name = 'Pindel'
+# no official release provided for version 0.2.5b9 (20160729),
+# therefore, pull last commit (20170508) to this version from master repo
+version = '0.2.5b9-20170508'
+
+homepage = 'http://gmt.genome.wustl.edu/packages/pindel/'
+description = """ Pindel can detect breakpoints of large deletions, medium sized
+ insertions, inversions, tandem duplications and other structural variants at single-based
+ resolution from next-gen sequence data. It uses a pattern growth approach to identify the
+ breakpoints of these variants from paired-end short reads. """
+
+toolchain = {'name': 'GCC', 'version': '6.4.0-2.28'}
+toolchainopts = {'openmp': True}
+
+source_urls = [
+    'https://github.com/genome/pindel/archive/',
+]
+sources = [{
+    'filename': 'v%(version)s.zip',
+    'download_filename': 'b706fba61c64a11fb1d3716d501fd2f4d8992e29.zip'
+}]
+patches = ['%(name)s-%(version)s_fix-ambig-abs-compil-err.patch']
+checksums = [
+    'd1dec65e157e2018fc827a90579caf27d096a9e3e798fd715b44470fe920361a',  # v0.2.5b9-20170508.zip
+    '216ae1145fd1568ac2187a89bf4b95c45d5c02272047a3dd5f9dfbede26c8fec',  # patch
+]
+
+# apply changes made in https://github.com/genome/pindel/pull/64 to fix compilation error
```

Comment:
@jdemeul This comment should be higher up, no?

Reply:
Argh, completely overlooked that one, apologies.
Thanks @boegel, should be fixed now!